app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
+    return sum([player.get_rating() for player in players])
```

Comment:
The `calculate_team_total_rating` function uses a list comprehension inside the `sum` function, which creates a redundant list. According to the checklist, you should use a generator expression instead to improve efficiency. Change `[player.get_rating() for player in players]` to `(player.get_rating() for player in players)`.